knowledge-base/textbox-add-icon.md

```diff
@@ -29,6 +29,9 @@ Is there anyway to do an 'addon' button with the TelerikTextBox?
 
 ## Solution
 
+>important **UI for Blazor 5.1.0** includes [Prefix and Suffix templates]({%slug common-features/input-adornments%}). Use that instead.
```

Comment:
This is another solution and must be separated from the rest of the content through a new heading. For example:
```
## Solution 1
UI for Blazor 5.1.0 provides ...
...

## Solution 2

If you use UI for Blazor 5.0.x or earlier,...

```